google-cloud-nio/src/main/resources/META-INF/native-image/com/google/cloud/google-cloud-nio/native-image.properties

```diff
@@ -0,0 +1,36 @@
+# Using META-INF/services with Native Image compilation results in
+# FileSystemProvider being initialized at build time. This results
+# CloudStorageFileSystemProvider and all the sub-classes referenced by
+# this class also being initialized at build time.
+Args = --initialize-at-build-time=com.google.cloud.storage.contrib.nio.CloudStorageFileSystemProvider,\
+  com.google.cloud.storage.contrib.nio.CloudStorageConfiguration,\
+  com.google.cloud.storage.contrib.nio.CloudStorageFileSystem,\
+  com.google.cloud.storage.contrib.nio.StorageOptionsUtil,\
+  com.google.cloud.storage.StorageException,\
+  com.google.cloud.storage.StorageOptions,\
+  com.google.cloud.storage.DefaultStorageRetryStrategy,\
+  com.google.api.client.http.OpenCensusUtils,\
+  com.google.api.client.http.GenericUrl,\
```

Comment:
(I'm curious about how you get the list)

If we don't have this `GenericUrl` line here, what kind of error do you get?

Reply:
Removing `GenericUrl` results in:
```
Error: Classes that should be initialized at run time got initialized during image building:
 com.google.api.client.http.GenericUrl was unintentionally initialized at build time. To see why com.google.api.client.http.GenericUrl got initialized use --trace-class-initialization=com.google.api.client.http.GenericUrl
```

And removing all the other classes results in a similar for the following classes:
 
```
--trace-class-initialization=com.google.api.client.json.gson.GsonParser$1,io.opencensus.trace.Tracing,io.opencensus.trace.EndSpanOptions,com.google.api.client.http.HttpRequest,org.threeten.bp.Duration,com.google.api.client.http.javanet.NetHttpTransport,com.google.auth.oauth2.OAuth2Utils,com.google.cloud.storage.StorageOptions,io.opencensus.contrib.http.util.CloudTraceFormat,com.google.auth.oauth2.DefaultCredentialsProvider,io.opencensus.trace.Status,io.opencensus.trace.AutoValue_EndSpanOptions,com.google.cloud.storage.DefaultStorageRetryStrategy,com.google.api.client.http.OpenCensusUtils,com.google.cloud.storage.StorageException,com.google.gson.stream.JsonReader,com.google.auth.oauth2.OAuth2Credentials,com.google.api.client.util.DateTime,com.google.api.client.util.escape.PercentEscaper,com.google.cloud.ExceptionHandler,io.opencensus.trace.BlankSpan,io.opencensus.trace.SpanContext,com.google.api.client.json.JsonParser$1,com.google.api.client.http.HttpTransport,com.google.api.client.http.HttpMediaType,com.google.auth.oauth2.GoogleCredentials,com.google.common.collect.RegularImmutableMap,io.opencensus.trace.Span,com.google.api.client.http.GenericUrl,com.google.cloud.ServiceOptions,com.google.api.client.util.Data,com.google.api.client.util.escape.CharEscapers,com.google.cloud.PlatformInformation
```

Reply:
I see. Thank you for checking.